Example/ExampleApp/RedViewController.swift

```diff
@@ -24,6 +24,7 @@ import PresenterKit
 final class RedViewController: UITableViewController {
 
     let cellId = "cell"
+    var dismissConfig: DismissButtonConfig?
```

Comment:
how about, instead of a public var, let's pass this in via `init` with a default of `nil` ?